packages/ui/primitives/data-table.tsx

```diff
@@ -115,7 +115,12 @@ export function DataTable<TData, TValue>({
               table.getRowModel().rows.map((row) => (
                 <TableRow key={row.id} data-state={row.getIsSelected() && 'selected'}>
                   {row.getVisibleCells().map((cell) => (
-                    <TableCell key={cell.id}>
+                    <TableCell
+                      key={cell.id}
+                      style={{
+                        width: cell.column.getSize(),
+                      }}
```

Comment:
I would have to assume that this is covered by specifying the size already, is that not the case?

Reply:
I didn't get your question, can you elaborate?

Reply:
I assume that tanstack table is already setting the width of the column using `cell.column.getSize()` so doing so in the style attribute here should be redundant.

Instead we should only need the manually specified size change that you had in the other part of this change.

Reply:
Okay, So are you suggesting to hardcode the width in this file?